docs/getting-started/index.mdx

```diff
@@ -6,174 +6,377 @@ pagination_next: http/index
-protect your app with a username and password like so:
+Now you can run ngrok again using the `--traffic-policy-file` flag.
 
 ```
-ngrok http http://localhost:8080 --basic-auth 'username:a-very-secure-password'
+ngrok http 8080 --url <YOUR_NGROK_DOMAIN> --traffic-policy-file auth.yaml
 ```
 
-<img width="450" height="450" src="/docs/img/docs/ngrok-basic-auth.png" />
+Anyone accessing your public endpoint will be prompted to log in with Google and
+only your account will be allowed to access it.
 
-ngrok supports many forms of authentication including:
+If you don't have a Google account or you want a simpler form of authentication,
+you can protect your app with the [basic authentication
+action](/docs/http/traffic-policy/actions/basic-auth.mdx) using a username and
+password.
+
+```yaml
+---
+on_http_request:
+  - actions:
+      - type: "basic-auth"
+        config:
+          credentials:
+            - "user:password1"
+```
 
-- [OAuth](/http/oauth/) (what we just used)
-- [Basic Auth](/http/basic-auth/) (what we just used)
-- [IP Restrictions](/http/ip-restrictions/)
-- [Webhook Verification](/http/webhook-verification/)
-- [Mutual TLS](/http/mutual-tls/)
-- [OpenID Connect](/http/openid-connect/)
-- [SAML](/http/saml/)
+Run your ngrok agent again:
 
-## Running ngrok Persistently
+```yaml
+ngrok http 8080 --url <YOUR_NGROK_DOMAIN> --traffic-policy-file auth.yaml
+```
 
-While this guide gets you started with ngrok, you might want your tunnel to keep running even after you close your terminal window. Here are a few options to achieve this:
+If you access your endpoint with a browser, it'll prompt you for the username
+and password. If you're putting an API service online and try to access it with
+`curl`, you'll get a `401 Unauthorized` response unless you base64 encode
+`user:password1` and add the result as a header:
 
-- Using Docker: Docker allows you to containerize ngrok for easy deployment and management. See the documentation for [Docker configuration](/using-ngrok-with/docker/).
+```bash
+curl --request GET \
+  --url http://<YOUR_NGROK_DOMAIN>/ \
+  --header 'Authorization: Basic ...`'
+```
 
-- Systemd service (Linux): For systemd-based systems, you can create a systemd service to automatically start and manage ngrok. Refer to the documentation for [Systemd service creation](/agent/#background-service).
-- Windows service (Windows): For Windows systems, you can create a Windows service to automatically start and manage ngrok. Refer to the documentation for [Background Service](/agent/#background-service).
-- Detaching a Tmux session (optional): If you're familiar with Tmux, you can detach your ngrok session from your terminal window. (Note: This is an advanced option). Refer to a guide on using [Tmux](https://www.digitalocean.com/community/tutorials/how-to-configure-tmux-on-a-cloud-server) for more information:
+ngrok supports many forms of authentication:
```

Comment:
I think if you changed this to be something more generic like "ways to protect access to your endpoint, then you can get away with ip restrictions and webhook verification.